Doc/c-api/weakref.rst

```diff
@@ -96,3 +96,10 @@ as much as it can.
    This iterates through the weak references for *object* and calls callbacks
    for those references which have one. It returns when all callbacks have
    been attempted.
+
+
+.. c:function:: void PyUnstable_Object_ClearWeakRefsNoCallbacks(PyObject *object)
+
+   Clears the weakrefs for *object* without calling the callbacks.
```

Comment:
Would you mind to document when and where this function is expected to be called? You described it here: https://github.com/capi-workgroup/decisions/issues/25#issue-2290152853 ("... Clear weakrefs again, but don't call callbacks ...").

I understood that it's mostly for deallocator functions, no?

I suggest to recommend using PyObject_ClearWeakRefs() in general.

Reply:
I've updated the docs